interop/test_utils.go

```diff
@@ -747,6 +751,14 @@ func DoSoakTest(tc testgrpc.TestServiceClient, serverAddr string, dopts []grpc.D
 			continue
 		}
 		fmt.Fprintf(os.Stderr, "soak iteration: %d elapsed_ms: %d succeeded\n", i, latencyMs)
+		if t != nil {
```

Comment:
Instead of creating a `time.Ticker` above, could you consider doing the following here?

```golang
if minTimeBetweenRPCs != 0 {
  <-time.After(minTimeBetweenRPCs)
}
```

With this approach, you wouldn't have to worry about stopping the ticker below.

Also, I'm not quite sure if `time.After()` returns immediately if it is passed a duration of `0`. If that is the case, you would need to check if the duration is non-zero either.

Reply:
Done, nice cleanup!

I made a slight change to your comment, which is to create the time.After channel before performing the RPC, and then wait on it after. This is because the flag is meant to control min time between RPC starts, rather than between finishes and starts.